packages/config/lib/options.ts

```diff
@@ -233,12 +249,6 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     defaultValue: 60000,
     validation: validate.isNumber,
     canUpdateDuringTestTime: true,
-  }, {
```

Comment:
FYI - This seems related to https://cypress-io.atlassian.net/browse/UNIFY-1292?filter=10047 which I'm working though & hoping to open a PR today for. Good news is we had the same change!

Reply:
Well, turns out my changes are not exactly related to the pluginsFile since they can normally only happen in the `pluginsFile` :D

Reply:
I removed this change since it was not part of this PR